sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -279,11 +279,12 @@ dependencies in one of two ways. Firstly, through buffer accessors that
 represent data dependencies between two command groups captured as nodes.
 Secondly, by using the `handler::depends_on()` mechanism inside a command group
 captured as a node. However, for an event passed to `handler::depends_on()` to
-create an edge, it must be an event returned from a queue
-submission captured by the same graph. Otherwise, a synchronous error will be
-thrown with error code `invalid`. `handler::depends_on()` can be
-used to express edges when a user is working with USM memory rather than SYCL
-buffers.
+create an edge, it must be of a specific type (see <<event-class-modifications,
+the "Event Class Modifications" section>> for more information). Using normal
+SYCL events from regular queue submissions is not allowed.
+`handler::depends_on()` can be used to express edges when a user is working with
+USM memory rather than SYCL buffers. For more information about event usage see
+<<event-class-modifications, the "Event Class Modifications" section>>. 
```

Comment:
For more information about event usage see <<event-class-modifications, the "Event Class Modifications" section>>. 

it is duplicate with line 282 and 283.

Reply:
Fixed thanks!